addons/web/static/src/frontend/error_notifications.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module */
```

Comment:
should remain in web/(...)/frontend/ ?

Reply:
I think we talked about having a public/ folder (for the database_manager at least)